src/plugins/discover/public/application/angular/doc_table/components/table_row/open.html

```diff
@@ -1,6 +1,6 @@
 <td ng-click="toggleRow()" data-test-subj="docTableExpandToggleColumn" class="kbnDocTableCell__toggleDetails">
   <button
-    class="euiButtonIcon euiButtonIcon--text"
+    class="euiButtonIcon euiButtonIcon--text euiButtonIcon--empty euiButtonIcon--xSmall"
```

Comment:
Why is this necessary? In Discover I can't see any styles applied based on these classes.

Reply:
The button(s) outlined in red use this template. Appears that Dashboard uses this Discover template.

![dashboard_preserve_layout](https://user-images.githubusercontent.com/2728212/112479490-16379a00-8d43-11eb-8b82-e49f909d933f.png)

Reply:
Yeah, I found them in Kibana, but I couldn't find any CSS rules attached to the new classes.

Reply:
Uses the `:not(.euiButtonIcon--empty)` selector to prevent certain default styles from being applied, rather than adding specific styles.